src/index.js

```diff
@@ -464,10 +454,9 @@ class AppRouter extends Component {
 We have to wrap our main app component in a generic <Route /> like this
 so we get the `location` in props
 */
-const AppRouterWrap = () => (
+const AppRouterWrap = () =>
```

Comment:
Can you revert this change and the one below? It created a conflict with the base-path.patch file which is used to make sure that react-router works properly on saferglobe.fi